spec/features/get_a_teacher_relocation_payment/trainee_route_completing_the_form_spec.rb

```diff
@@ -16,7 +16,8 @@
   end
 
   describe "navigating forward" do
-    it "submits an application" do
+    # Well be removing this feature spec in the next commit
+    xit "submits an application" do
```

Comment:
See this pr https://github.com/DFE-Digital/claim-additional-payments-for-teaching/pull/2905